src/test/groovy/com/synacy/gradprogram/spock/exercise/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,171 @@
+        println "Number of Items in Cart: ${cart.getItems().size()}"
+        eligibleForDiscount
+    }
+
+    def "Apply discount if the Cart is Eligble for discount"(){
+        given:
+        def items = [
+                new Item("555 Tuna", 50.00, ItemType.FOOD),
+                new Item("Pork Chop", 150.00, ItemType.FOOD),
+                new Item("Charger", 100.00, ItemType.GADGET),
+                new Item("Vanz Tshirt", 160.00, ItemType.CLOTHING),
+                new Item("Oven", 800.00, ItemType.APPLIANCE),
+                new Item("Fan", 200.00, ItemType.APPLIANCE),
+        ]
+        when:
+        double originalTotalCost = orderingService.calculateTotalCostOfCart(cart = new Cart(UUID.randomUUID(), items))
+        orderingService.applyDiscountToCartItems(cart)
+
+        then:
+        orderingService.isCartEligibleForDiscount(cart) == true
+        double discountedTotalCost = (originalTotalCost * 0.90)
+
+        println "Total Price: $originalTotalCost"
+        println("")
+        println "Total Discounted Price: $discountedTotalCost"
+    }
+
+    def "Should not Apply discount if the Cart is not Eligble for discount"(){
+        given:
+        def items = [
+                new Item("555 Tuna", 50.00, ItemType.FOOD),
+                new Item("Pork Chop", 150.00, ItemType.FOOD),
+                new Item("Charger", 100.00, ItemType.GADGET),
+                new Item("Vanz Tshirt", 160.00, ItemType.CLOTHING),
+                new Item("Oven", 800.00, ItemType.APPLIANCE)
+        ]
+        when:
+        double originalTotalCost = orderingService.calculateTotalCostOfCart(cart = new Cart(UUID.randomUUID(), items))
+        orderingService.applyDiscountToCartItems(cart)
+
+        then:
+        orderingService.isCartEligibleForDiscount(cart) == false
+        double discountedTotalCost = originalTotalCost
+
+        println "Total Price: $originalTotalCost"
+        println("")
+        println "Total Discounted Price: $discountedTotalCost"
+    }
+
+    def "Should create an order when the cart does not contain food items and canContainFood is true"(){
+        given:
+        def items = [
+                new Item("555 Tuna", 50.00, ItemType.FOOD),
+                new Item("Pork Chop", 150.00, ItemType.FOOD),
+                new Item("Charger", 100.00, ItemType.GADGET),
+                new Item("Vanz Tshirt", 160.00, ItemType.CLOTHING),
+                new Item("Oven", 800.00, ItemType.APPLIANCE)
+        ]
+        cart = new Cart(UUID.randomUUID(), items)
+
+        String recipientName = "Ernest Pogi"
+        String recipientAddress = "Cebu City"
+        boolean canContainFood = true
+
+        when:
+        Order order = orderingService.createAnOrder(cart, recipientName, recipientAddress, canContainFood)
+
+        then:
+        order != null
+        order.getTotalCost() == orderingService.calculateTotalCostOfCart(cart)
```

Comment:
Same here, lets assert using an actual value instead of calling another service method.